src/protocol/protocol.h

```diff
@@ -68,6 +68,7 @@ PROTODEF(PROTOCOL_USBHID, TX_MODULE_LAST,  NULL, USBHID_Cmds, "USBHID")
 PROTODEF(PROTOCOL_CRSF, TX_MODULE_LAST,  NULL, CRSF_Cmds, "CRSF")
 #ifndef MODULAR
 PROTODEF(PROTOCOL_SBUS, TX_MODULE_LAST,  NULL, SBUS_Cmds, "SBUS")
+PROTODEF(PROTOCOL_PXX, TX_MODULE_LAST,  NULL, PXXOUT_Cmds, "PXX")
```

Comment:
we can support this define in devo7e as there are bytes to use.

Reply:
I this thought sbus and pxx modules are larger than 4kb

Reply:
The pxx module is less than 1.5K, and sbus is less because it lacks telemetry.  The issue with pxx is the soft serial support.  It uses interrupts so whatever is the issue with the uart code is likely to affect pxx telemetry too.